llvm/unittests/Transforms/IPO/CMakeLists.txt

```diff
@@ -11,6 +14,19 @@ add_llvm_unittest(IPOTests
   LowerTypeTests.cpp
   WholeProgramDevirt.cpp
   AttributorTest.cpp
+  DynamicallyLoadedInliningAdvisor.cpp
   )
 
+add_llvm_library(DAdvisor MODULE BUILDTREE_ONLY DAdvisor.cpp)
```

Comment:
We need more strict checks (e.g., are we running on windows?) like in this file: https://github.com/IBricchi/llvm-project/blob/ib/dlib-advisor-work/llvm/unittests/Passes/CMakeLists.txt